lib/src/rpc.ml

```diff
@@ -24,3 +29,27 @@ let index_put =
   let bin_query = [%bin_type_class: string * string * string list]
   and bin_response = [%bin_type_class: unit] in
   Async.Rpc.Rpc.create ~name:"put_index" ~version:0 ~bin_query ~bin_response
+
+let metadata_put =
+  let bin_query = [%bin_type_class: string * string]
```

Comment:
What do those `string`s stand for? I think this module might benefit from some type aliases and/or comments.